tests/integration/test_render.py

```diff
@@ -194,3 +199,36 @@ def test_background_image(self):
         image_path = '/resource/drag-and-drop-v2/public/img/triangle.png'
         self.assertTrue(bg_image.get_attribute("src").endswith(image_path))
         self.assertEqual(bg_image.get_attribute("alt"), 'This describes the target image')
+
+    def test_borders_hidden(self):
+        self.load_scenario()
+        zones = self._get_zones()
+        for index in range(len(zones)):
```

Comment:
@agaylard Nit: I think the version using `enumerate(zones)` that you were using earlier is much more readable. If you name a variable `_` or `dummy`, Pylint will not produce an error about it being unused (cf. [this answer](http://stackoverflow.com/a/10107410/1199226) on SO), which means that this:

```python
for index, dummy in enumerate(zones, start=1):
```

should pass the quality checks.

The `start=1` part let's you use the simpler `.format(index)` instead of `.format(index+1)`.